File: Sihwan-Kim/baekjoon/BOJ_14681.java
package BOJ;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

/**
 * @see <a href="https://www.acmicpc.net/problem/14681">
 * https://www.acmicpc.net/problem/14681
 * </a>
 */
public class BOJ_14681 {
    public static void main(String[] args){
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int[] arr = new int[2];
		for (int i = 0; i < 2; i++) {
			StringTokenizer st = new StringTokenizer(br.readLine());
			arr[i] = Integer.parseInt(st.nextToken());
		}
		
		if(arr[0] > 0 && arr[1] > 0) {
			System.out.println(1);
		} else if(arr[0] < 0 && arr[1] > 0) {
			System.out.println(2);
		} else if(arr[0] < 0 && arr[1] < 0) {
			System.out.println(3);
		} else if(arr[0] > 0 && arr[1] < 0) {
			System.out.println(4);
		}
	}
    }
}

File: Hyoil-Cha/baekjoon/BOJ_100/BOJ_14681.py
import sys

"""
    feat(Hyoil-Cha): [BOJ, 14681] 사분면 고르기

    https://www.acmicpc.net/problem/14681
"""

def solution() :

    while True:
        x = int(input())
        if -1000 <= x and x <= 1000 and x != 0 :
            break
    
    while True:
        y = int(input())
        if -1000 <= y and y <= 1000 and y != 0 :
            break
    
    if x > 0 and y > 0 :
        print(1)
    elif x < 0 and y >0 :
        print(2)
    elif x < 0 and y < 0 :
        print(3)
    else:
        print(4)

solution()

